build.gradle

```diff
@@ -158,10 +158,10 @@ test {
   systemProperties['hudson.model.DownloadService.noSignatureCheck'] = 'true'
   // override default timeout
   systemProperties['jenkins.test.timeout'] = 300
-  ignoreFailures = gradle.ciBuild
+  ignoreFailures = gradle.isCi
```

Comment:
I think we only want to ignore test failures on Jenkins and not on Teamcity.

Reply:
Thinking about this some more, shouldn't we just remove that completely and have it fail always? Why has this been added in the first place? To make the Jenkins build green even though there were test failures ?

Reply:
Jenkins builds have three possible outcomes: red, yellow and green/blue. The yellow state is what GE calls a verification failure. In order for Jenkins to show a verification failure for Gradle builds, the build needs to succeed and there need to be failing tests. That is why I ignored failures when running on CI.

Reply:
Understood. Changed it.